src/Data/Bifunctor/Linear.hs

```diff
@@ -4,23 +4,45 @@
 {-# LANGUAGE NoImplicitPrelude #-}
 {-# LANGUAGE TypeOperators #-}
 
-module Data.Bifunctor.Linear where
+-- | This module provides Bifunctor and related classes.
+--
+-- == 'Bifunctor'
+--
+-- Use a bifunctor instance to map functions over structures that hold
+-- two data types and could have a functor instance over either one.
```

Comment:
I think I've written this before, but “hold two data types” is not very meaningful terminology, we need to find something better.